models/Issue.js

```diff
@@ -6,14 +6,9 @@ var issueSchema = mongoose.Schema({
 	title: String,
 	content: String,
 	author_id: String,
-	date_created: Number
+	date_created: Date
```

Comment:
We just need to be mindful that when we provide the data out to the iOS application, that this is converted into Randy's preferred format: a String that is the "Full Month Name", "2 digit day", and "4 digit year", ie: "June 09 2015" or "May 22 2015"